feat(audio): Nighttime Music Tracks + New Music Manager

Zepalesque opened the pull request:
Nighttime music tracks (which only play at night, and are the _only_ ones that play at night), plus the slightly pitched-down version of Welcome to Paradise heard in the Aether II (both GoTV and Highlands), along with Brise for the daytime music selection.

closes issue #61

---

I agree to the Contributor License Agreement (CLA).

bconlon1 commented on src/main/resources/assets/aether/sounds.json:
```diff
@@ -1,8 +1,9 @@
     "sounds": [
       {
-        "name": "aether:music/aether1",
+        "name": "aether_genesis:music/aether1_lowpitch",
```
I would say add this track to the list instead of replacing it.

Zepalesque committed:
Base music files + updated forge

- The older Forge version was incompatible with the Aether Beta 3

Zepalesque committed:
feat: Night Music Manager

update: Base music files + updated forge

- The older Forge version was incompatible with the Aether Beta

feat: Fully Functioning Music Manager

- ported over from the Aether: Redux

docs: removed a leftover TODO comment from the Aether: Redux

bconlon1 reviewed (changes requested):
Merge conflicts have to be resolved.

Zepalesque commented:
ok, all of the issues have been resolved (I also made all songs except welcome to paradise have a weight of 2, and each of the two welcome to paradise versions have a weight of 1, so that welcome to paradise doesn't play twice as often as other songs, which could have gotten repetitive otherwise)

Zepalesque committed:
Merge branch 'night-music-manager' of https://github.com/Zepalesque/The-Aether-Genesis into night-music-manager

Zepalesque committed:
Merge branch 'temp-merge' into night-music-manager

Zepalesque committed:
refactor(audio): Both versions of Welcome to Paradise now play

- weight settings have been modified so that Welcome to Paradise still plays an equal amount, choosing one of the two versions of the song

bconlon1 commented on src/generated/resources/assets/aether_genesis/lang/en_us.json:
```diff
@@ -15,6 +15,7 @@
   "block.aether_genesis.storm_aercloud": "Storm Aercloud",
   "block.aether_genesis.stripped_skyroot_log_wall": "Stripped Skyroot Log Wall",
   "block.aether_genesis.stripped_skyroot_wood_wall": "Stripped Skyroot Wood Wall",
```
Typo at "preveent"

Zepalesque replied:
ok, it's fixed now

bconlon1 commented on src/generated/resources/assets/aether_genesis/lang/en_us.json:
```diff
@@ -15,6 +15,7 @@
   "block.aether_genesis.storm_aercloud": "Storm Aercloud",
   "block.aether_genesis.stripped_skyroot_log_wall": "Stripped Skyroot Log Wall",
   "block.aether_genesis.stripped_skyroot_wood_wall": "Stripped Skyroot Wood Wall",
+  "config.aether.client.client.night_music_tracks": "Adds some nice night tracks to the Aether's music selection. Also disables the default music manager for the Aether, to preveent overlap",
```
The typo is still there.

Zepalesque replied:
uhh, that’s odd, I’ll try to fix it at some point soon-ish

Zepalesque replied:
oh wait I forgot to rerun the datagen, so that’s why it’s still there, I can probably rerun that at some point

Zepalesque replied:
Actually wait, I can just edit in the browser since the datagen code already has the fix. It should be fixed now, hopefully

Zepalesque committed:
docs: fixed typo

- preveent -> prevent

Zepalesque committed:
fix: fixed typo in json file

Zepalesque committed:
fix: fixed config localization in datagen

Zepalesque committed:
fix: fixed the config localization in the json file

bconlon1 committed:
Merge branch '1.19.4-develop' into night-music-manager

Aether-Team-Bot committed:
chore: Trigger build

bconlon1 reviewed (approved):
lgtm
